packages/next/src/client/components/react-dev-overlay/ui/components/errors/error-overlay-footer/error-feedback/error-feedback.tsx

```diff
@@ -63,16 +64,26 @@ export function ErrorFeedback({ errorCode, className }: ErrorFeedbackProps) {
             </a>
           </p>
           <button
-            aria-label="Mark as helpful"
-            onClick={() => handleFeedback(true)}
+            aria-disabled={disabled ? 'true' : undefined}
+            aria-label={
```

Comment:
should we add a `title` explaining why it's disabled, for non screen-reader users?

Reply:
Right, that way we also don't need to change the label since the title will be treated as a description on elements that are labelled by other means (here: `aria-label`).